.github/workflows/e2e-tests.yaml

```diff
@@ -13,11 +13,11 @@ on:
 
 jobs:
   run-in-k8s:
-    name: Run in Kubernetes 1.26 (${{ github.event.inputs.target }})
+    name: Run in Kubernetes latest (${{ github.event.inputs.target }})
     environment: E2E
     env:
-      FLC_NAMESPACE: dto-k8s-ondemand
-      FLC_ENVIRONMENT: dto-k8s-1-26
+      FLC_NAMESPACE: dto-daily
+      FLC_ENVIRONMENT: dto-k8s-latest-flc
```

Comment:
somewhat unrelated to this specific PR, but is there a reason we are not using a [matrix](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#example-using-a-single-dimension-matrix) to reuse the steps? It seems only the `FLC_ENVIRONMENT` is different between `run-in-k8s` and `run-in-ocp` jobs

Reply:
I just wrote a user story today about this, you can check it in the epic (I have no access to Jira now)